### ISSUE TITLE
Incorrect error message

### ISSUE THREAD

**Issue by bosborne** [jekyll/jekyll#4306]
Version 2.4.0.  I have a Liquid template file that's used to make JSON, it's called "search.json". It had been working well for me, then I added a few new Markdown files and started to get this error:
```
5933 >       Regenerating: 1 file(s) changed at 2016-01-03 11:59:17   Liquid Exception: invalid byte sequence in UTF-8 in search.json
...error:
             Error: invalid byte sequence in UTF-8
             Error: Run jekyll build --trace for more information.
```

So I re-typed the search.json file by hand, but still got the error, which didn't make sense. So I ran ```tr``` over these new Markdown files like this:
```
tr -cd '^[:print:]' < howtos/Scrapbook_HOWTO.md > Scrapbook_HOWTO.md
```

That fixed the problem. Meaning the error message was wrong, the problem was not in the template "search.json" file, the problem was in the input Markdown files.

BTW, Jekyll is great, really enjoying it. Supremely easy to set up and run.

**Comment by parkr** [comment]
> Version 2.4.0

Can you reproduce this with v3.0? If not, then this can be closed. If you're beholden to `github-pages`, keep an eye on: https://github.com/github/pages-gem/pull/175

**Comment by bosborne** [comment]
Fixed in 3.01, let's close this.
